doc/UsersGuide/source/IdealizedTC.rst

```diff
@@ -0,0 +1,82 @@
+.. role:: raw-html(raw)
+    :format: html
+
+.. _idealized-tc:
+
+**************************************
+Idealized Tropical Cyclone Case
+**************************************
+
+The idealized, regional tropical cyclone case is derived from the I-HAFS configuration (:cite:t:`Wang2024`) and is designed to support controlled studies of tropical cyclone dynamics and forecast development. This configuration removes real-world data assimilation and ocean coupling, focusing solely on atmospheric forecasts using idealized inputs.
+
+Initial and lateral boundary conditions (ICs/LBCs) are derived from a large-scale, idealized global FV3-based atmospheric forecast. The initial vortex is constructed using the Reed and Jablonowski (2011) method, introducing a weak, balanced storm into an environment favorable for rapid intensification. The preprocessing system generates fixed distributions of geography-related variables and constructs the ICs/LBCs from ``tcvitals`` and GRIB input files.
+
+The configuration mirrors the operational HAFS structure but simplifies terrain and surface properties. It includes:
+
+- Preprocessing to set up the forecast and nest domains  
+- Optional vortex initialization  
+- FV3-based forecast integration  
+- Postprocessing to generate GRIB2 and ATCF output files
```

Comment:
again, this is available in IHAFS, but here we utilize pre-generated ICs/LBCs. i just want that to be clear for users. 

in the ufs hsd framework, they can adjust fv3 settings in the input namelist, try different physics suites, adjust computational parameters, but we only provide one set of ICs/LBCs for this idealized case (pre/post processing mentioned here is for IHAFS). 

however, we do include a plotting script in ufs-wm/tests-dev/test_cases/utils called `plot_tc.sh` that allows users to use UFS-WM generated grib files to plot 10m wind for the TC, as well as generate a gif from these wind plots. could you add a snippet about this plotting option, please?